packages/application-extension/schema/kernel-subshells.json

```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Kernel Subshells",
```

Comment:
I think we need to consolidate kernel settings in a single place. There is also "Kernel dialogs" which only has one setting.

Reply:
Also "Running Terminals and Kernels" with just one setting.

Reply:
Should I merge them all in some "Terminals and Kernels" settings?

Reply:
I've moved the subshells settings closer to the existing kernel settings

Reply:
> That would be a breaking change, wouldn't it be? Since user settings would stop working.

No, adding settings to existing sections is fine. Also, that particular one was not even released. I think this should block release but more necessarily this PR.